server/container_exec.go

```diff
@@ -24,7 +26,7 @@ func (s *Server) Exec(ctx context.Context, req *pb.ExecRequest) (resp *pb.ExecRe
 func (s StreamService) Exec(containerID string, cmd []string, stdin io.Reader, stdout, stderr io.WriteCloser, tty bool, resize <-chan remotecommand.TerminalSize) error {
 	c, err := s.runtimeServer.GetContainerFromShortID(containerID)
 	if err != nil {
-		return fmt.Errorf("could not find container %q: %v", containerID, err)
+		return status.Errorf(codes.NotFound, err.Error())
```

Comment:
To still keep the information of the containerID
```suggestion
		return status.Errorf(codes.NotFound, "could not find container %q: %v", containerID, err)
```

Reply:
The same with the change in container_attach.go too.

Reply:
Agreed.  Humans will depend on that when the error is much lower level - "ENOMEM: failed" lacks the context we would want